spring-integration-core/src/main/java/org/springframework/integration/channel/FluxMessageChannel.java

```diff
@@ -52,11 +52,11 @@
 
 	private volatile boolean upstreamSubscribed;
 
-	public ReactiveChannel() {
+	public FluxMessageChannel() {
 		this(DirectProcessor.create());
 	}
 
-	public ReactiveChannel(FluxProcessor<Message<?>, Message<?>> processor) {
+	public FluxMessageChannel(FluxProcessor<Message<?>, Message<?>> processor) {
```

Comment:
Sorry, I missed somehow discussion.
Would you mind explaining to me one more time?
So, what is wrong to request just raw `Processor` and `wrap()` internally if that is still required?
Thanks